examples/pwa-sw.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+self.addEventListener('install', event => {
+});
+
+self.addEventListener('fetch', event => {
+  // TODO(dvoytenko): use cache, implement one-behind.
+  if (event.request.url.indexOf('amp.max.html') != -1) {
+    // Override response with the shell unless the leaf document is explicitly
+    // requested.
+    const isDirectFetch = event.request.headers.get('AMP-Direct-Fetch') == '1';
+    if (!isDirectFetch) {
+      event.respondWith(fetch('/pwa'));
```

Comment:
Don't you still, even without the cache, want to initiate the request here? The browser cache should do the right thing.